rescript.md

```diff
@@ -34,4 +34,4 @@ let rec len = (myList: list<'a>) =>
 
 ## Other considerations:
 
-1. Prefer using uncurried functions over curried (i. e. _`reduceU` over `reduce`_ in Belt.List*)*, it will help the compiler to produce cleaner error messages in cases when there is a parameters mismatch, which happens quite often during refactoring. Another benefit is that it works faster.
+1. Prefer using uncurried functions over curried (i. e. _`reduceU` over `reduce`_ in Belt.List*)*, it will help the compiler to produce cleaner error messages in cases when there is a parameters mismatch, which happens quite often during refactoring. Another benefit is that it works faster. [Nice didn't know this]
```

Comment:
Yeah, grabbed it from the reference style-guide I was looking at. Wasn't something I thought about before.

It is just much nicer writing curried functions.